src/query/xasl_cache.c

```diff
@@ -1998,6 +1997,7 @@ xcache_cleanup (THREAD_ENTRY * thread_p)
       count = 0;
       /* Collect candidates for cleanup. */
       lf_hash_create_iterator (&iter, t_entry, &xcache_Ht);
+      gettimeofday (&current_time, NULL);
```

Comment:
I use this variable for finding the difference between the current time and last time used (line 2008), I think it is useful :)